pyccl/massfunction.py

```diff
@@ -0,0 +1,68 @@
+from . import halos as hal
+from .pyutils import deprecated
+
+
+@deprecated(hal.MassFunc)
+def massfunc(cosmo, halo_mass, a, overdensity=200):
+    """Halo mass function, dn/dlog10M.
+    Args:
+        cosmo (:obj:`Cosmology`): Cosmological parameters.
+        halo_mass (float or array_like): Halo masses; Msun.
+        a (float): scale factor.
+        overdensity (float): overdensity parameter (default: 200)
+    Returns:
+        float or array_like: Halo mass function; dn/dlog10M.
+    """
+    mdef = hal.MassDef(overdensity, 'matter')
+    mf_par = cosmo._config_init_kwargs['mass_function']
+    if mf_par == 'tinker10':
+        mf = hal.MassFuncTinker10(cosmo, mdef)
+    elif mf_par == 'tinker':
+        mf = hal.MassFuncTinker08(cosmo, mdef)
+    elif mf_par == 'watson':
+        mf = hal.MassFuncWatson13(cosmo, mdef)
+    elif mf_par == 'shethtormen':
+        mf = hal.MassFuncSheth99(cosmo)
+    elif mf_par == 'angulo':
+        mf = hal.MassFuncAngulo12(cosmo)
+
+    return mf.get_mass_function(cosmo,
+                                halo_mass,
+                                a)
+
+
+@deprecated(hal.HaloBias)
+def halo_bias(cosmo, halo_mass, a, overdensity=200):
+    """Tinker et al. (2010) halo bias
+    Args:
+        cosmo (:obj:`Cosmology`): Cosmological parameters.
+        halo_mass (float or array_like): Halo masses; Msun.
+        a (float): Scale factor.
+        overdensity (float): Overdensity parameter (default: 200).
+    Returns:
+        float or array_like: Halo bias.
+    """
+    mdef = hal.MassDef(overdensity, 'matter')
+    mf_par = cosmo._config_init_kwargs['mass_function']
+    if mf_par == 'tinker10':
+        bf = hal.HaloBiasTinker10(cosmo, mdef)
+        return bf.get_halo_bias(cosmo,
+                                halo_mass,
+                                a)
+    else:
+        raise NotImplementedError("No b(M) fitting function implemented for "
```

Comment:
This probably raised a CCLError before.

Reply:
If we're not supporting the sheth&tormen ones anymore, I think we should check here.

Reply:
The errors raised are also part of the API.

Reply:
OK, for the reasons above I'll just change this for a CCLError then.

Reply:
Done